schedule/schedule-appengine/src/main/client/jasify/services/multipass.service.js

```diff
@@ -1,7 +1,7 @@
 (function (angular) {
     angular.module('jasifyComponents').factory('Multipass', multipass);
 
-    function multipass(Endpoint) {
+    function multipass(Jasify, Endpoint) {
```

Comment:
We should introduce a new service that holds the `fetchId()` method.  The objective is to replace `Endpoint` and we are still depending on that method.  Or do you think it should be embedded in every call on the generated code?